docs/virtual_machines/tekton_tasks.md

```diff
@@ -115,11 +115,18 @@ KubeVirt-specific Tekton tasks, which are focused on:
 #### Wait for Virtual Machine Instance Status
 - wait-for-vmi-status - Waits for a VMI to be running.
 
+#### Modify Windows iso
+- modify-windows-iso-file - modifies windows iso (replaces prompt bootloader with no-promt bootloader) and replaces original iso 
+  in PVC with updated one.  This helps with automation of windows which requires EFI - the prompt bootloader will not continue with 
```

Comment:
`This helps with automated installation of Windows in EFI boot mode. By default Windows uses a prompt bootloader, which will not continue with the boot process until a key is pressed. By replacing it with the non-prompt bootloader no key press is required to boot into the Windows installer.`?